scripts/breed_bird_survey.py

```diff
@@ -12,11 +12,16 @@
 import urllib.request, urllib.parse, urllib.error
 import zipfile
 from decimal import Decimal
+try:
+    from retriever.lib.templates import Script
```

Comment:
The first line of the try and except blocks looks the same and so I think it can be moved to the main import block.